tests/core/CoreTestBase.py

```diff
@@ -4,7 +4,7 @@
 
 
 class CoreTestBase(unittest.TestCase):
-    def execute_run(self, bears, executor=None):
+    def execute_run(self, bears, cache=None, executor=None):
```

Comment:
Same here.

Reply:
Oh please file an issue actually, this commit attempts to change something different than style.

Reply:
Because this component already exists.